systems/inventory/accounting/pkg/config.go

```diff
@@ -0,0 +1,50 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * Copyright (c) 2023-present, Ukama Inc.
+ */
+
+package pkg
+
+import (
+	"time"
+
+	uconf "github.com/ukama/ukama/systems/common/config"
+	metric "github.com/ukama/ukama/systems/common/metrics"
+)
+
+const (
+	GaugeType = "gauge"
```

Comment:
Why do we have metrics here and it is not being used. I think you can remove this.